udata/templates/search.html

```diff
@@ -16,48 +16,39 @@
 {% set datasets_search_url = url_for('datasets.list') %}
 {% set reuses_search_url = url_for('reuses.list') %}
 {% set organizations_search_url = url_for('organizations.list') %}
+{% set breadcrum_class='search-bar'%}
 
 {% block breadcrumb_bar %}
 <!-- Nav tabs -->
-<ul class="nav nav-tabs search-tabs">
+<ul class="nav nav-tabs search-tabs nav-tabs--search-tabs">
     {% if datasets %}
         <li class="active" data-tab="datasets">
             <a href @click.prevent="setTab('datasets')">
-                {{ _('Datasets')}}
-                <span class="badge">{{ datasets.total }}</span>
+                {{ ngettext('%(num)d dataset', '%(num)d datasets',datasets.total)}}
             </a>
         </li>
     {% endif %}
     {% if reuses %}
         <li{% if not datasets %} class="active"{% endif %} data-tab="reuses">
             <a href @click.prevent="setTab('reuses')">
-                {{ _('Reuses') }}
-                <span class="badge">{{ reuses.total }}</span>
+                {{ ngettext('%(num)d reuse', '%(num)d reuses', reuses.total) }}
             </a>
         </li>
     {% endif %}
     {% if organizations %}
         <li{% if not datasets and not reuses %} class="active"{% endif %} data-tab="organizations">
             <a href @click.prevent="setTab('organizations')">
-                {{ _('Organizations') }}
-                <span class="badge">{{ organizations.total }}</span>
+                {{ ngettext('%(num)d organization', '%(num)d organizations', organizations.total) }}
             </a>
         </li>
     {% endif %}
     {% if territories %}
         <li{% if not datasets and not reuses and not organizations %} class="active"{% endif %} data-tab="territories">
             {% set total=territories|length %}
-            {% if total==1 %}
-                <a href="{{ territories.0.url }}">
-                    {{ _('Territory:') }}
-                    <span class="badge">{{ territories.0.name }}</span>
-                </a>
-            {% else %}
-                <a href @click.prevent="setTab('territories')">
-                    {{ _('Territories') }}
-                    <span class="badge">{{ total }}</span>
-                </a>
-            {% endif %}
+            <a href="{% if total==1 %}{{ territories.0.url }}{% endif %}" {% if total>1%}@click.prevent="setTab('territories'){% endif %}>
```

Comment:
Given we already are in a `{% if territories %}` (ie. total > 0), I think the '{% endif %} {% if total >1 %}` can be even more simplfied in a `{% else %}` ;)